unit_tests/OrthoTree/ortho_tree.cpp

```diff
@@ -0,0 +1,66 @@
+//
+// unit test of morton_codes.h
+
+#include "gtest/gtest.h"
+
+#include "OrthoTree/OrthoTree.h"
+#include "OrthoTree/Types.h"
+
+#include <algorithm>
+#include <cstdint>
+#include <bitset>
+#include <vector>
+#include <array>
+
+using namespace ippl;
+
+TEST(OrthoTreeTest, Constructor)
+{
+    static constexpr size_t Dim = 3;
+    ippl::OrthoTree<Dim> tree(5, 10, ippl::BoundingBox<Dim>({ 0.0, 0.0, 0.0 }, { 1.0, 1.0, 1.0 }));
+    EXPECT_EQ(1, 1);
```

Comment:
This check is nonsense.
Maybe check for some parameters that are set on initialization if we want to test anything here

Reply:
Is removed